src/components/ChartContextMenu.svelte

```diff
@@ -48,6 +49,34 @@
 
   const dateFormatter = timeFormat('%Y-%m-%d');
   const timeFormatter = timeFormat('%H:%M:%S');
+
+  const getTelemetryPath = () => {
+    if ($store.probe.type === 'histogram') {
+      return ['main', 'parent'].includes($store.productDimensions.process)
+        ? `payload.histograms.${$store.probe.name}`
+        : `payload.processes.${$store.productDimensions.process}.histograms.${$store.probe.name}`;
+    }
+    return undefined;
+  };
+
+  const table =
+    $store.productDimensions.channel === 'nightly'
+      ? 'main_nightly'
+      : 'main_1pct';
+
+  const getComparisonViewinSTMO = () =>
+    $store.product === 'firefox'
+      ? 'https://sql.telemetry.mozilla.org/queries/82247/source?' +
+        `&p_Table=telemetry.${table}` +
+        `&p_Probe=${getTelemetryPath()}` +
```

Comment:
What happens when `getTelemetryPath()` returns `undefined`? I assume then for scalars the `href` below will be set to some invalid link and then you can click `View Comparison in STMO` but the link won't work. We may need some check to only show the button if the value is a histogram.